### PR TITLE
Add README for notebooks dir

### PR DESCRIPTION
**Pull request by hemajv**
## Related Issues and Dependencies

#88 

## This introduces a breaking change

- [ ] Yes
- [X] No

## This Pull Request implements

Adding a README to describe the notebooks directory structure for easier understanding

### CODE REVIEW

File: notebooks/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+### Notebooks
+
+This folder contains various reproducible and interactive Jupyter notebooks created for the project. These notebooks are also available for viewing on our public [JupyterHub instance](https://jupyterhub-opf-jupyterhub.apps.cnv.massopen.cloud/hub/login).
+The notebook organizational structure is described in detail below.
+
+#### Data Sources
+
+The notebooks defined in the [`data-sources`](notebooks/data-sources) folder, explore and analyze the [Sippy](https://github.com/openshift/sippy) and [TestGrid](https://github.com/GoogleCloudPlatform/testgrid) data sources.
```
Comment (chauhankaranraj) [inline notebooks/README.md:8]: The link to `data-sources` (and also links to other directories and notebooks) seem to give a 404. Is that just me or is there an extra `notebooks` in the link path?
```suggestion
The notebooks defined in the [`data-sources`](data-sources) folder, explore and analyze the [Sippy](https://github.com/openshift/sippy) and [TestGrid](https://github.com/GoogleCloudPlatform/testgrid) data sources.
```

**Review by chauhankaranraj — CHANGES_REQUESTED** [review]

**Review by oindrillac — CHANGES_REQUESTED** [review]
Love the write-up @hemajv! :heart: 

The relative URLs in this document give 404s. I believe it is because they are `notebooks/data-sources/Sippy/sippy_failure_correlation.ipynb` instead of `data-sources/Sippy/sippy_failure_correlation.ipynb` since the README.md is a part of the `notebooks` directory.

File: notebooks/README.md (hunk 1 above)
Comment (hemajv) [reply notebooks/README.md]: ah yes, links updated :+1:

**Comment by hemajv** [comment]
> Love the write-up @hemajv! 
> 
> The relative URLs in this document give 404s. I believe it is because they are `notebooks/data-sources/Sippy/sippy_failure_correlation.ipynb` instead of `data-sources/Sippy/sippy_failure_correlation.ipynb` since the README.md is a part of the `notebooks` directory.

fixed the relative links, should work now :+1:

**Review by oindrillac — APPROVED** [review]
:ship: :ship:

**Review by aakankshaduggal — APPROVED** [review]
Looks good 👍

**Comment by chauhankaranraj** [comment]
> fixed the relative links, should work now

Looks great! :ship:

**Review by chauhankaranraj — APPROVED** [review]

**Review by MichaelClifford — APPROVED** [review]

**Pull request merged**